### ISSUE TITLE
Skater library development is suspended

### ISSUE THREAD

**Issue by jamesmyatt** [christophM/interpretable-ml-book#124]
Development of the Skater library is currently suspended according to 
https://github.com/oracle/Skater/issues/269#issuecomment-459624034, backed up by the lack of Github activity from project members. It might be a good idea to replace the references to it, or look for alternatives to mention.

**Comment by christophM** [comment]
Thanks! Will look for other libraries to recommend.

**Comment by kteavery** [comment]
There's a PermutationImportance pip package that I've used before. It might do the trick for the 5.5 Feature Importance section: https://pypi.org/project/PermutationImportance/.

**Comment by jamesmyatt** [comment]
Permutation importance is in eli5: https://github.com/TeamHG-Memex/eli5
